src/lib/api/feed-manip.ts

```diff
@@ -55,16 +55,15 @@ export class FeedViewPostsSlice {
     }
     this._feedPost = feedPost
     this._reactKey = `slice-${post.uri}-${
-      feedPost.reason?.indexedAt || post.indexedAt
+      feedPost.reason && 'indexedAt' in feedPost.reason
+        ? feedPost.reason.indexedAt
+        : post.indexedAt
     }`
     if (feedPost.post.uri === FALLBACK_MARKER_POST.post.uri) {
       this.isFallbackMarker = true
       return
     }
-    if (
-      !AppBskyFeedPost.isRecord(post.record) ||
-      !AppBskyFeedPost.validateRecord(post.record).success
-    ) {
+    if (!AppBskyFeedPost.isValidRecord(post.record)) {
```

Comment:
I don't think we need these validations here, but they were already in place and I wanted to double check if they're necessary. Left them for now, will revisit before this merges.